src/blocks/image/edit.js

```diff
@@ -19,14 +19,19 @@ function UAGBImageEdit( props ) {
 		attributes,
 		name,
 		attributes: { UAGHideDesktop, UAGHideTab, UAGHideMob },
-		deviceType
+		deviceType,
+		context
```

Comment:
Always use trailing comma.

```suggestion
		context,
```

Reply:
This coding practice has been implemented [in this PR](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3283).

This conversation can be resolved once that PR is merged.